lib/rag-engines/index.ts

```diff
@@ -41,10 +41,7 @@ export class RagEngines extends Construct {
     const tables = new RagDynamoDBTables(this, "RagDynamoDBTables");
 
     let sageMakerRagModels: SageMakerRagModels | null = null;
-    if (
-      props.config.rag.engines.aurora.enabled ||
-      props.config.rag.engines.opensearch.enabled
-    ) {
+    if (props.config.llms.enableSagemakerModels) {
```

Comment:
This should be checking `crossEncodingEnabled` and not `enableSageMakerModels`

Reply:
ok, but won't that be confusing that crossEncodingEnabled is driving the Sagemaker models instead of the config `props.config.llms.enableSagemakerModels` which is specific for sagemaker models?

Reply:
You are right and `props.config.llms.enableSagemakerModels` is better